web/packages/teleport/src/Welcome/NewCredentials/types.ts

```diff
@@ -60,6 +66,13 @@ export type NewCredentialsProps = UseTokenState & {
     username,
     onSubmit,
   }: QuestionnaireProps) => ReactElement;
+
+  // support for E's invite collaborators at onboarding
+  displayInviteCollaborators?: boolean;
+  setDisplayInviteCollaborators?: (bool: boolean) => void;
```

Comment:
oh so it is a setter, i think setting a default to false was a mistake then? (see previous comment)

Reply:
Yup, I'll set it to null by default instead (per the type signature I suppose).